include/swift/AST/Types.h

```diff
@@ -799,7 +799,13 @@ class alignas(1 << TypeAlignInBits) TypeBase {
   
   /// Check if this is a nominal type defined at the top level of the Swift module
   bool isStdlibType();
-  
+
+  /// Check if this is a CGFloat type from `CoreGraphics` module.
```

Comment:
Can you update the comment to reflect that you're also checking for this type in swift-corelibs-foundation?

Reply:
Sure!

Reply:
✅